examples/fps_cur_ridgecv_pipeline.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+from matplotlib import pyplot as plt
+
+from sklearn.pipeline import Pipeline
+from sklearn.linear_model import RidgeCV
+from sklearn.datasets import load_boston
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+
+from skcosmo.feature_selection import SimpleFPS, SimpleCUR
+
+scaler = StandardScaler()
```

Comment:
It is great to have examples, but I thing they could be made more useful by adding a few comments describing what is going from 10000ft above

Reply:
Added a little comment!